### PR TITLE
Fix watcher log level description in documentation

### PR DESCRIPTION
**Pull request by xinnjie**
# Changes
Fix wrong description about log level in `docs/DEVELOPMENT.md`

/kind documentation

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you review them:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [Tested your changes locally](https://github.com/tektoncd/results/blob/main/docs/DEVELOPMENT.md) (if this is a code change)
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user-facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contain the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/ok-to-test

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ See [test/e2e/README.md](/test/e2e/README.md)
 You can change log level controllers for development purpose. Edit configmap:
 
 ```sh
-kubectl edit cm config-logging -n tekton-pipelines
```
Comment (xinnjie) [inline docs/DEVELOPMENT.md:282]: Miss `tekton-results-` here.  `config/components/metadata/kustomization.yaml` set this prefix

**Comment by xinnjie** [comment]
@alan-ghelardi @dibyom Take a look please.

**Review by adambkaplan — APPROVED** [review]
/approve

/lgtm

Thanks for these clarifications!

**Pull request merged**